src/dstack/_internal/cli/commands/ps.py

```diff
@@ -36,10 +36,17 @@ def _register(self):
             help="Watch statuses of runs in realtime",
             action="store_true",
         )
+        self._parser.add_argument(
+            "-n",
+            "--last",
+            help="Show only the last N runs",
```

Comment:
(nit) It is not very intuitive that `-n` also enables displaying stopped runs, so let's at least document it.
```suggestion
            help="Show only the last N runs. Implies --all",
```